docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -17,11 +17,27 @@ For a conceptual overview of blob storage, see [the .NET guide for blob storage]
 
 To use this guide, you must first [create an Azure storage account](/azure/storage/common/storage-account-create). You also need your storage access key for this account.
 
-## Create an F# Script and Start F# Interactive
+## Create an F\# script and start F\# interactive
 
 The samples in this article can be used in either an F# application or an F# script. To create an F# script, create a file with the `.fsx` extension, for example `blobs.fsx`, in your F# development environment.
 
-Next, use a [package manager](package-management.md) such as [Paket](https://fsprojects.github.io/Paket/) or [NuGet](https://www.nuget.org/) to install the `WindowsAzure.Storage` and `Microsoft.WindowsAzure.ConfigurationManager` packages and reference `WindowsAzure.Storage.dll` and `Microsoft.WindowsAzure.Configuration.dll` in your script using a `#r` directive.
+### How to execute scripts
+
+F# Interactive, `dotnet fsi`, can be launched interactively, or it can be launched from the command line to run a script. The command-line syntax is
+
+```.NET CLI
+> dotnet fsi [options] [ script-file [arguments] ]
+```
+
+### Add packages in a script
+
+Next, use `#r` `nuget:package name` to install the `Azure.Storage.Blobs` and `Azure.Storage.Blobs.Models` and `Azure.Storage.Blobs.Specialized` packages.Such as
+
+```fsharp
+> #r "nuget: Azure.Storage.Blobs"
+> #r "nuget: Azure.Storage.Blobs.Models"
+> #r "nuget: Azure.Storage.Blobs.Specialized"
```

Comment:
There's only an `Azure.Storage.Blobs` package.  The others are just namespaces in that package.  I'm assuming this would fail if you ran it?  Maybe instead you wanted:
```fsharp
#r "nuget: Azure.Storage.Blobs"
open Azure.Storage.Blobs
open Azure.Storage.Blobs.Models
open Azure.Storage.Blobs.Specialized
```